<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterTacheComposite</class>
 <widget class="QDialog" name="ajouterTacheComposite">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter une tache composite</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nom"/>
         </item>
         <item>
          <widget class="QGroupBox" name="contient">
           <property name="title">
            <string>Contient</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QListWidget" name="listeContient"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Disponibilité</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="disponibilite">
           <property name="displayFormat">
            <string>dddd d MMMM yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Echéance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="echeance">
           <property name="displayFormat">
            <string>dddd d MMMM yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="constraitsToogle">
           <property name="text">
            <string>Contrainte de précédence</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="boxconst">
           <property name="title">
            <string>Tache précédente</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QListWidget" name="listeTachesPred"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ajouterTacheComposite</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>constraitsToogle</sender>
   <signal>clicked(bool)</signal>
   <receiver>boxconst</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disponibilite</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>echeance</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ajouterTacheComposite</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
